src/bitmessageqt/addressvalidator.py

```diff
@@ -122,23 +153,28 @@ def validate(self, s, pos):
         if address is None:
             addressGeneratorQueue.put(('createChan', 4, 1, str_chan + ' ' + str(passPhrase), passPhrase, False))
         else:
-            addressGeneratorQueue.put(('joinChan', addBMIfNotPresent(address), str_chan + ' ' + str(passPhrase), passPhrase, False))
+            addressGeneratorQueue.put(
+                ('joinChan', addBMIfNotPresent(address),
+                 str_chan + ' ' + str(passPhrase), passPhrase, False))
```

Comment:
I would use `format` here, I was told using `+` to join strings should be avoided.